modules/apps/depot/depot-web/src/main/resources/META-INF/resources/item/selector/disabled.jsp

```diff
@@ -0,0 +1,19 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+Esto no está habilitao
```

Comment:
@AliciaGarciaGarcia This kind of thing shouldn't be visible in the commit history. Can you perform a bit of commit engineering to clean up the PR?